Rebuild-Black-Business/RBB-Website
# Feature/general page layout

# Describe your PR

Adds basic layouts to several pages. Also adjusts the Business filters to match late conversations with design / data teams.

Also adds responsive styles to the ContactCards along with a couple of semantic fixes.

## Pages/Interfaces that will change

/businesses
/allies
/about

### Screenshots / video of changes

https://gyazo.com/3e7ba27555d657d23ea1ddac6c7329b1


## Steps to test

1. Visit the /businesses page and ensure that you can filter by businesses in need, or all businesses
2. Visit the /allies page and ensure each ally has a ResultCard
3. Visit the /about page and ensure the ContactCards are responsive
